malariagen_data/ag3.py

```diff
@@ -6549,6 +6549,323 @@ def displayTapNodeData(data):
             run_params["width"] = width
         return app.run_server(**run_params)
 
+    def _fst_gwss(
+        self,
+        contig,
+        window_size,
+        sample_sets,
+        cohort1_query,
+        cohort2_query,
+        site_mask,
+        cohort_size,
+        random_seed,
+    ):
+
+        ds_snps1 = self.snp_calls(
+            region=contig,
+            sample_query=cohort1_query,
+            sample_sets=sample_sets,
+            site_mask=site_mask,
+            cohort_size=cohort_size,
+            random_seed=random_seed,
+        )
+
+        ds_snps2 = self.snp_calls(
+            region=contig,
+            sample_query=cohort2_query,
+            sample_sets=sample_sets,
+            site_mask=site_mask,
+            cohort_size=cohort_size,
+            random_seed=random_seed,
+        )
+
+        print("computing allele counts for cohort 1")
+        gt1 = allel.GenotypeDaskArray(ds_snps1["call_genotype"].data)
+        ac1 = gt1.count_alleles()
+
+        print("computing allele counts for cohort 2")
+        gt2 = allel.GenotypeDaskArray(ds_snps2["call_genotype"].data)
+        ac2 = gt2.count_alleles()
+
+        pos = ds_snps1["variant_position"].values
+
+        fst = allel.moving_hudson_fst(ac1, ac2, size=window_size)
+        x = allel.moving_statistic(pos, statistic=np.mean, size=window_size)
+
+        results = dict(x=x, fst=fst)
+
+        return results
+
+    def fst_gwss(
+        self,
+        contig,
+        window_size,
+        cohort1_query,
+        cohort2_query,
+        site_mask,
```

Comment:
```suggestion
        site_mask="gamb_colu_arab",
```

Suggest defaulting to safest site mask.